gitlab_integration/README.md

```diff
@@ -1,16 +1,34 @@
 # Gitlab integration research
 
+## Project integrations
+
+It allow you to integrate GitLab with other applications. They are a bit like plugins in that they allow a lot of freedom in adding functionality to GitLab.
+
+Just like Github Apps but with some limited functionality. This will be a very easy way for any projects to use the Packit service (eg in https://gitlab.gnome.org)
+
+So just by clicking one button, packit service will be active for the project - 
+
+![packit service example](./gitlab-packit.png)
```

Comment:
This is a customizable settings page, isn't it? (So a gui version of our config file.)

Can we react on a change? (So we can e.g. create a PR with the config change or add a badge,...)

Reply:
>This is a customizable settings page, isn't it? (So a gui version of our config file.)

Yes

>Can we react on a change? (So we can e.g. create a PR with the config change or add a badge,...)

What do you mean by creating a PR with the config change?

Reply:
That was just an example reaction from our side when the user saves the settings.
- Get the values from the settings page and update the config-file with those values and create a merge-request with the changes.
- It can be misleading to users when I am thinking about that. (They can be in a weird situation when they don't merge the merge-request.)

We can maybe find a better use-case.

Reply:
So how about getting the information in the events itself and ditching the config file completely. 

There should be a way when user updates the settings page we can send us an event with updated settings, But for this, we will have to work more on the GitLab side.